templates/patient/card/photo.html.twig

```diff
@@ -7,5 +7,10 @@
         {{ imageWidget(p, patientPhotoCategoryName)}}
```

Comment:
should the above be 
```
{% if photos %}
    {% for p in photos %}
        {{ imageWidget(p, patientPhotoCategoryName)}}

Reply:
The function returns the first id in the category. Not an array like patient photo.
Curious as to why. Perhaps trying to limit the size of the widget.
I'm guessing this has been the case since beginning unless Robert changed the old.

Reply:
could be changed from `patientIDCategoryID` to `patientPhotoCategoryName`

if user wanted more insurance cards then would be have to be on 1 image, maybe someday it could be moved down into the insurance card

Reply:
I could modify id cards to more than one and maybe limit to three if everyone thinks it is useful. what do you think?

Reply:
sounds great

Reply:
for some reason I haven't found widget only allows two photos!

Reply:
while testing noticed that we're not saving any user settings with the `initiallyCollapsed` thing

Reply:
yep I noticed awhile back.

Reply:
okay set up to limit id to 3. sound reasonable for max number?

Reply:
Removed the annoying uuid of file name. Of no use to user

![image](https://user-images.githubusercontent.com/1859985/193479623-1bf98211-012c-4cca-9741-8e5ee50d200f.png)